server/src/test/java/org/elasticsearch/index/store/StoreTests.java

```diff
@@ -485,11 +486,16 @@ public void testRecoveryDiff() throws IOException, InterruptedException {
         List<Document> docs = new ArrayList<>();
         for (int i = 0; i < numDocs; i++) {
             Document doc = new Document();
-            doc.add(new StringField("id", "" + i, random().nextBoolean() ? Field.Store.YES : Field.Store.NO));
-            doc.add(new TextField("body",
-                TestUtil.randomRealisticUnicodeString(random()), random().nextBoolean() ? Field.Store.YES : Field.Store.NO));
-            doc.add(new SortedDocValuesField("dv", new BytesRef(TestUtil.randomRealisticUnicodeString(random()))));
+            final Field.Store stringFieldStored = random().nextBoolean() ? Field.Store.YES : Field.Store.NO;
+            doc.add(new StringField("id", "" + i, stringFieldStored));
+            final String textFieldContent = TestUtil.randomRealisticUnicodeString(random());
+            final Field.Store textFieldStored = random().nextBoolean() ? Field.Store.YES : Field.Store.NO;
+            doc.add(new TextField("body", textFieldContent, textFieldStored));
+            final String docValueFieldContent = TestUtil.randomRealisticUnicodeString(random());
+            doc.add(new BinaryDocValuesField("dv", new BytesRef(docValueFieldContent)));
```

Comment:
This block of changes is mostly for extra logging, the one semantic change here is `SortedDocValuesField` becoming `BinaryDocValuesField`. I struggled to update the original `SortedDocValuesField` for some reason, I'm not too familiar with this API so maybe I missed something?